src/ConnectedMode/Migration/RuleSetCleaner.cs

```diff
@@ -18,32 +18,36 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
```

Comment:
The GitHub side-by-side diff looks a bit weird for this file, so you might want to ignore the "before".

I moved the `MigrationProgress` class to another file and added this new `RuleSetCleaner` class. For some reason, GitHub is showing the unrelated new file as a modification of the deleted file.